infrastructure/messaging/application/config/settings.py

```
@@ -0,0 +1,87 @@
+import os
+from dataclasses import dataclass
+from typing import List, Optional
```

Flake8 (7.3.0)

[error] 'typing.List' imported but unused

(F401)